### PR TITLE
Added trip properties

### PR DESCRIPTION
**Pull request by datanel**

### CODE REVIEW

File: src/objects.rs [hunk 1]
```diff
@@ -704,6 +704,46 @@ pub struct Transfer {
...
+#[derive(Serialize, Deserialize, Debug, Derivative, PartialEq)]
+#[derivative(Default)]
+pub enum TransportType {
+    #[derivative(Default)]
+    #[serde(rename = "0")]
+    Regular,
+    #[serde(rename = "1")]
+    ExclusiveSchool,
+    #[serde(rename = "2")]
+    Mixed,
```
Comment (datanel) [inline src/objects.rs]: better names? @prhod @TeXitoi

Comment (TeXitoi) [reply src/objects.rs]: I'm not found of these names, but I can't find anything better now.

Comment (prhod) [reply src/objects.rs]: For the `Mixed` value, it's meaning is not clear when read alone. What about `RegularAndSchool` ?

Comment (datanel) [reply src/objects.rs]: ok for RegularAndSchool

**Pull request merged**